### PR TITLE
Fix product tests resource processing

### PR DESCRIPTION
**Pull request by arhimondr**
Apply maven resource pluging filtering only to `presto.env` file
instead of to all the presto-product-tests resources.

Maven resource filtering is used to replace properties
placeholders e.g. `${project.version}` in the module resource files.
In our case we use this feature to substitute the actual presto version
in `presto.env` file which is used for running product tests on travis.
The problem is that the `presto-product-tests` module contains
many binary files with the test tables data. Resource plugin with filtering
enabled always tries to parse resources as text files and writes
the parsed result to the target even if no placeholders has been replaced.
This operation breaks the binary files.

### CODE REVIEW

File: presto-product-tests/pom.xml [hunk 1]
```diff
@@ -83,8 +83,18 @@
...
         <resources>
             <resource>
-                <directory>${project.basedir}/src/main/resources</directory>
+                <directory>src/main/resources</directory>
                 <filtering>true</filtering>
+                <includes>
+                    <include>presto.env</include>
+                </includes>
+            </resource>
+            <resource>
```
Comment (kokosing) [inline presto-product-tests/pom.xml]: do you need to exclude rest?

File: presto-product-tests/pom.xml [hunk 2]
```diff
@@ -83,8 +83,18 @@
     <build>
         <resources>
             <resource>
-                <directory>${project.basedir}/src/main/resources</directory>
+                <directory>src/main/resources</directory>
```
Comment (kokosing) [inline presto-product-tests/pom.xml]: why did you remove `${project.basedir}`?

**Comment by kokosing** [comment]
looks good modulo comments

File: presto-product-tests/pom.xml (hunk 1 above)
Comment (arhimondr) [inline presto-product-tests/pom.xml]: We need to run resource plugin with filtering enabled only for the `presto.env` file, and we need to run it for all the other resources with filtering disables. Besides the replacing properties this plugin also does the copy of the resources to the `target`.

File: presto-product-tests/pom.xml (hunk 2 above)
Comment (arhimondr) [inline presto-product-tests/pom.xml]: Path is relative to `${project.basedir}` by default. I don't think that we need to specify it explicitly.

**Comment by martint** [comment]
Merged, thanks!
